src/main/java/com/agiletec/aps/system/services/user/IUserManager.java

```diff
@@ -89,6 +89,15 @@
      */
     public void changePassword(String username, String password) throws EntException;
 
+    /**
```

Comment:
Hey @BrenoQVDS , 
Could you please translate this javadoc in English ?
Thanks

Reply:
I will do that, sorry.
I already changed, but I will commit in my next PR so we don't have to wait for all the tests here, ok?